src/main/java/rx/schedulers/Schedulers.java

```diff
@@ -32,7 +35,14 @@
     private final Scheduler ioScheduler;
     private final Scheduler newThreadScheduler;
 
-    private static final Schedulers INSTANCE = new Schedulers();
+    private static Schedulers INSTANCE;
+
+    private static synchronized Schedulers getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new Schedulers();
+        }
+        return INSTANCE;
+    }
```

Comment:
`shutdown()` is currently a static method. Should `current.shutdown()` be just `shutdown()` or should `shutdown()` be changed to an instance method?